numbers-csharp/ConvertString2Int/ConvertString2Int/Benchmark.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConvertString2Int
+{
+    public static class Benchmark
+    {
+        const int formatOffsetLeft = -19;
+        const int formatOffsetRight = 13;
+
+        static string BenchmarkIntParse(string strValue, int reps)
+        {
+            int number;
+            var watch = new Stopwatch();
+            watch.Start();
+            for (int i = 0; i < reps; i++)
+            {
+                number = int.Parse(strValue);
+            }
+            watch.Stop();
```

Comment:
You'll need to remove the StopWatch after the suggestions in the article, so read them first.